QUARKUS-2492.md

```diff
@@ -13,11 +13,13 @@ Test development will focus on
 - Quarkus JDBC transactions based on programmatic approach
 - Quarkus JDBC transactions based on lambda approach
 - Verify commons transactions states as "begin", "commit" and "rollback"
-- Verify classic and reactive flavors
+- Reactive flavor is out of scope, currently only classic approach is supported
 - The following databased will be covered: Oracle, Mysql, MariaDB, Mssql, Postgresql
```

Comment:
Also pls update this line to use proper DB name, not the "common" variants of the names ... e.g. use MySQL, PostgreSQL